website/source/docs/agent/checks.html.markdown

```diff
@@ -37,18 +37,19 @@ There are five different kinds of checks:
   limited to roughly 4K. Responses larger than this will be truncated.
 
 * TCP + Interval - These checks make an TCP connection attempt every Interval
-  (e.g. every 30 seconds) to the specified IP/hostname and port. The status of
-  the service depends on whether the connection attempt is successful (ie - the
-  port is currently accepting connections). If the connection is accepted, the
-  status is `success`, otherwise the status is `critical`. In the case of a
-  hostname that resolves to both IPv4 and IPv6 addresses, an attempt will be
-  made to both addresses, and the first successful connection attempt will
-  result in a successful check. This type of check should be preferred over a
-  script that uses `netcat` or another external process to check a simple socket
-  operation. By default, TCP checks will be configured with a request timeout
-  equal to the check interval, with a max of 10 seconds. It is possible to
-  configure a custom TCP check timeout value by specifying the `timeout` field
-  in the check definition.
+  (e.g. every 30 seconds) to the specified IP/hostname and port. If no hostname
```

Comment:
This is the change right here... the rest are my editor trying to be "helpful" and remove trailing spaces.